test/use-swr-infinite.test.tsx

```diff
@@ -572,4 +572,55 @@ describe('useSWRInfinite', () => {
     fireEvent.click(screen.getByText('data:'))
     await screen.findByText('data:')
   })
+  it('should mutate a cache with mumutateInfinite', async () => {
+    let count = 0
+    function Page() {
+      const { data } = useSWRInfinite<string, string>(
+        index => `page-test-12-${index}`,
+        key => createResponse(`${key}:${++count}`)
+      )
+      return <div>data:{data}</div>
+    }
+
+    render(<Page />)
+    screen.getByText('data:')
+
+    await screen.findByText('data:page-test-12-0:1')
+
+    await act(() => mutateInfinite(index => `page-test-12-${index}`))
+    await screen.findByText('data:page-test-12-0:2')
+
+    await act(() =>
+      mutateInfinite(index => `page-test-12-${index}`, 'local-mutation', false)
```

Comment:
users have the `keyLoader` definition, at the same time they also have the `createCache` to get the `mutate` function, then combining both of them can also achieve the similar approach?

Reply:
Do you mean like `mutate(getInfiniteKey(getKey), 'local-mutation', false)`?

Reply:
yes, that would be nice!